demo-app/src/lib/uqi-sources.ts

```diff
@@ -0,0 +1,13 @@
+import createTrinoUqiClient from '@/lib/trino-uqi-client'
+
+const sources = {
+  'trino-demo': {
+    config: {
+      auth: 'trino',
+      server: 'http://trino.orb.local:8080',
```

Comment:
Are you using docker instead of https://orbstack.dev/ then?

I just pushed a commit to fix it for everyone else for now. dea66c32868d433617645829bc1ce6f0d9866cb7

Reply:
Yea just docker. TIL about orbstack!